MantidQt/CustomInterfaces/test/ReflEventPresenterTest.h

```diff
@@ -26,14 +26,26 @@ class ReflEventPresenterTest : public CxxTest::TestSuite {
 
   ReflEventPresenterTest() {}
 
-  void test_slicing_options() {
+  void test_get_slicing_values() {
     MockEventView mockView;
     ReflEventPresenter presenter(&mockView);
 
-    EXPECT_CALL(mockView, getTimeSlices())
+    EXPECT_CALL(mockView, getTimeSlicingValues())
         .Times(Exactly(1))
-        .WillOnce(Return("MultiDetectorAnalysis"));
-    presenter.getTimeSlicingOptions();
+        .WillOnce(Return("5"));
+    presenter.getTimeSlicingValues();
+
+    TS_ASSERT(Mock::VerifyAndClearExpectations(&mockView));
+  }
+
+  void test_get_slicing_type() {
+    MockEventView mockView;
+    ReflEventPresenter presenter(&mockView);
+
+    EXPECT_CALL(mockView, getTimeSlicingType())
+        .Times(Exactly(1))
+        .WillOnce(Return("UniformEven"));
```

Comment:
Just for information, if the value returned by the mocked view is not going to be used, you can just do:

```
EXPECT_CALL(mockView, getTimeSlicingType()).Times(Exactly(1));
```